code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -346,7 +348,6 @@
 	mag_type = /obj/item/ammo_box/magazine/recharge/ctf
 	desc = "This looks like it could really hurt in melee."
 	force = 50
-	flags = NODROP | DROPDEL
```

Comment:
Keep the DROPDEL please, not having ammo or guns on the floor of the arena is a design choice of sorts.